static/app/utils/gettingStartedDocs/node.ts

```diff
@@ -124,6 +136,66 @@ export function getDefaulServerlessImports({
   return imports;
 }
 
+export function getNodeRunCommandSnippet(): string {
+  return `# If you are using CommonJS (CJS)
+node --require ./instrument.js app.js
```

Comment:
m: I thought we got rid of this and only suggest `--import` in case of ESM.

Reply:
This is an unused function right now. I can delete it.